index.src.html

```diff
@@ -586,6 +587,62 @@ <h3 id="feature-detect">Feature Detection</h3>
   Documents should not behave differently or not based on whether the UA
   implements this specification or not - all documents should assume it does.
 
+  <h3 id="premission-prompt">Permission Prompt</h3>
+
+  Following the discussions in
+  [[Issue#23](https://github.com/WICG/private-network-access/issues/23)], the
+  private network access permission prompt is introduced to relax mixed content
+  checks.
+
+  The goal of the permission is to allow communication from public websites to
+  local network servers over HTTP, which would otherwise be prevented by the
+  <a href="#secure-context-restriction">secure context restriction</a> and mixed
+  content checks. Migrating private network servers to HTTPS has indeed proven
+  to be often difficult, even sometimes impossible.
+
+  We propose adding a new parameter to the `fetch()` options bag. For example:
```

Comment:
The explainer was written at a time when we were not sure of the path forward, hence the tentative language. In the spec, I think we can talk about the prompt as a fact in the present tense:

```suggestion
  A new parameter is added to the `fetch()` options bag:
```